python/src/Curve_doc.i.in

```diff
@@ -34,3 +34,42 @@ Examples
 >>> # Create the curve
 >>> myCurve = ot.Curve(data, 'blue', 'solid', 2, '')
 >>> myGraph.add(myCurve)"
+
+// ---------------------------------------------------------------------
+
+%define OT_Curve_FillBetween_doc
+"Fill an area between two curves.
+
+Parameters
+----------
+dataX : 2-d sequence of floats of dimension 1, or sequence of float Points
+    First coordinates common to the curves.
+dataY1 : 2-d sequence of floats of dimension 1, or sequence of float Points
+    Second coordinates of the first curve.
+dataY2 : 2-d sequence of floats of dimension 1, or sequence of float Points
+    Second coordinates of the second curve.
+
+Returns
+-------
+polygon : :class:`~openturns.Polygon`
+    Polygon which fills the surface between the two curves.
+
+Examples
+--------
+>>> import openturns as ot
+>>> from math import pi, sin
+>>> # Generate the data for the curve to be drawn
+>>> data = [[-3 * pi + 6 * pi * i / 100, sin(-3 * pi + 6 * pi * i / 100)] for i in range(100)]
+>>> # Create the curve
+>>> myCurve = ot.Curve(data, 'blue', 'solid', 2, '')
+>>> # Create the polygon to fill between the curve and the x-axis
+>>> myPolygon = ot.Curve.FillBetween([[xy[0]] for xy in data], [[xy[1]] for xy in data], [[0.]] * len(data))
```

Comment:
The line is very long and is not displayed.
![image](https://github.com/openturns/openturns/assets/31351465/7b107345-3fc3-4c82-99a5-7b9067894ee4)


```suggestion
>>> xData = [[xy[0]] for xy in data]
>>> y1Data = [[xy[1]] for xy in data]
>>> y2Data = [[0.0]] * len(data)
>>> myPolygon = ot.Curve.FillBetween(xData, y1Data, y2Data)
```